index.html

```diff
@@ -1,275 +1,267 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="Stylesheet" href="Style.css" />
-	<link href="https://fonts.googleapis.com/css?family=Poppins" rel="Stylesheet" />
-    <title>Project "Mobile porfolio "</title>
-    
-  </head>
-  <body>
-
-    <header class="menu-containers">
 
-			<div class="logo"> 
-			
-        <a class="logo" href="#"><img src="Images/Logo Placeholder.png"></a>
-	        </div>
-			
-		<nav id="dell">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse seconde project</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] Hey, your styles are not showing because you're using lowerCase for the letter `s` while it is in uppercase in the directory. Kindly fix this issue to make your styles visible

- [ ] Also, kindly add the missing background image in the contact section to match the Figma design as much as you can.